.github/workflows/pre-commit.yaml

```diff
@@ -0,0 +1,23 @@
+name: pre-commit
+on:
+  push:
+
+    branches:
+      - 'fix/*'
+      - 'fixes/*'
+      - 'feature/*'
+      - 'features/*'
+      - 'hotfix/*'
+      - 'hotfixes/*'
+      - 'release/*'
+      - 'releases/*'
+      - 'chore/*'
+      - 'chores/*'
+      - 'automatic*'
+
+jobs:
+  pre-commit:
+    uses: boldlink/terraform-module-support/.github/workflows/pre-commit.yaml@1.0.0
```

Comment:
should this not be updated to `...@main`?

Reply:
Yes, the best place to update to main though is `terraform-module-template` repo, to also take effect in this repo during the next scheduled update